### PR TITLE
feature: convert level 32 files to typescript

### PR DESCRIPTION
**Pull request by DDDDDanica**
## Pre-merge author checklist

- [x] I've clearly explained:
  - [x] What problem this PR is solving
  - [x] How this problem was solved
  - [x] How reviewers can test my changes
- [ ] Sufficient automated test coverage has been added

## Pre-merge reviewer checklist

- [ ] Manual testing (e.g. pull and build branch, run in browser, test code being changed)
- [ ] PR is linked to the appropriate GitHub issue
- [ ] **IF** this PR fixes a bug in the release milestone, add this PR to the release milestone

If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.

### CODE REVIEW

File: ui/helpers/constants/design-system.ts [hunk 1]
```diff
@@ -54,6 +54,7 @@ export const COLORS = {
   GOERLI_INVERSE: 'goerli-inverse',
   SEPOLIA_INVERSE: 'sepolia-inverse',
 };
+
 export const BACKGROUND_COLORS = pick(COLORS, [
```
Comment (brad-decker) [inline ui/helpers/constants/design-system.ts:58]: These are technically enums, and we didn't convert them to such. But if we do that we are going to have *a lot * of changes. Probably best  to do that in a separate PR down the road.

Comment (brad-decker) [reply ui/helpers/constants/design-system.ts:58]: Note: ui/helpers/constants/routes.js are NOT enums, they are constants because the main object is a MAP of keys to values, versus something we'd be referencing.

For example COLORS from design system we use *everywhere* a Typography component can only ever have one COLORS assigned, so COLORS should be `Color` and the keys should be lowerCamelCased `Color.backgroundAlternative` @georgewrmarshall @garrettbear just a heads up, we aren't doing this change here.

**Review by brad-decker — DISMISSED** [review]
LGTM good work @DDDDDanica

File: ui/helpers/constants/design-system.ts (hunk 1 above)
Comment (DDDDDanica) [reply ui/helpers/constants/design-system.ts:58]: I agree we should have a separate MR following to cover the colors. will work on that next !

**Review by brad-decker — APPROVED** [review]

**Review by ryanml — APPROVED** [review]
